ext/bcmath/tests/bcmod.phpt

```diff
@@ -6,15 +6,274 @@ bcmath
 bcmath.scale=0
 --FILE--
 <?php
-echo bcmod("11", "2"),"\n";
-echo bcmod("-1", "5"),"\n";
-echo bcmod("8728932001983192837219398127471", "1928372132132819737213"),"\n";
-echo bcmod("3.5", "4", 1),"\n";
-echo bcmod("1071", "357.5"),"\n";
+
+$dividends = ["15", "-15", "1", "-9", "14.14", "-16.60", "0.15", "-0.01"];
+$divisors = array_merge($dividends, ["15151324141414.412312232141241", "-132132245132134.1515123765412", "141241241241241248267654747412", "-149143276547656984948124912", "0.1322135476547459213732911312", "-0.123912932193769965476541321"]);
+$scales = [0,10];
+
+foreach($scales as $scale) {
+	foreach($dividends as $dividend) {
+    	echo "Number \"$dividend\" (scale $scale)\n";
+		foreach($divisors as $divisor) {
+			echo str_pad($dividend, 6, ' ', STR_PAD_LEFT), " mod ", str_pad($divisor, 30), ' = ', bcmod($dividend, $divisor, $scale),"\n";
```

Comment:
Right the behaviour of "mod" is a remainder and not actually a module but that's an overarching problem in PHP.